### PR TITLE
Volodymyr Kulbashevsky's java service skeleton homework

### PR DESCRIPTION
**Pull request by Coolbash**
GL Android procamp HomeWork03: java service skeleton (service + getter app + setter app).

### CODE REVIEW

File: 03-JavaServiceSkeleton/Setter/app/src/main/java/com/my1st/setter/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                }
+            }
+        });
+    }
+
+    @Override
+    protected void onDestroy()
+    {
+        //unbinding from our service
+        if(my1st_service != null)
```
Comment (BulldozerBSG) [inline 03-JavaServiceSkeleton/Setter/app/src/main/java/com/my1st/setter/MainActivity.java]: Not a good idea to wrap the call unbindService() in a pointer check to null
From docs: If bindService() return true, you should later call unbindService(ServiceConnection) to release the connection.
In this case, just remove the check

File: 03-JavaServiceSkeleton/Getter/app/src/main/java/com/my1st/getter/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                refresh();
+            }
+        });
+    }
+
+    @Override
+    protected void onDestroy()
+    {
+        //unbinding from our service
+        if(my1st_service != null)
```
Comment (BulldozerBSG) [inline 03-JavaServiceSkeleton/Getter/app/src/main/java/com/my1st/getter/MainActivity.java]: Not a good idea to wrap the call unbindService() in a pointer check to null
From docs: If bindService() return true, you should later call unbindService(ServiceConnection) to release the connection.
In this case, just remove the check

**Review by BulldozerBSG — CHANGES_REQUESTED** [review]
Please use android code style

**Comment by Coolbash** [comment]
> 
> 
> Please use android code style

What is Android code style? Maybe I forgot something but I don't recall anybody told us anything about Android code style...

**Comment by BulldozerBSG** [comment]
> > Please use android code style
> 
> What is Android code style? Maybe I forgot something but I don't recall anybody told us anything about Android code style...

The code is written in a different style:
The curly brace opens on the same line, then on a new one.
In places there is a space between operators and brackets, in places there is no.
Therefore, I proposed to bring everything to one style of source code

**Comment by BulldozerBSG** [comment]
Examples of code style:
https://source.android.com/setup/contribute/code-style
or
https://google.github.io/styleguide/javaguide.html

**Comment by BulldozerBSG** [comment]
As an option, use automatic code formatting in the Android Studio
https://developer.android.com/studio/intro/?gclid=EAIaIQobChMIvqzkx-vP6QIVRoGyCh3XCwebEAAYASAAEgLOMvD_BwE&gclsrc=aw.ds#style_and_formatting

**Comment by Coolbash** [comment]
> As an option, use automatic code formatting in the Android Studio
> https://developer.android.com/studio/intro/?gclid=EAIaIQobChMIvqzkx-vP6QIVRoGyCh3XCwebEAAYASAAEgLOMvD_BwE&gclsrc=aw.ds#style_and_formatting

done

**Review by BulldozerBSG — APPROVED** [review]
Reviewed. No comments

**Pull request merged**